### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by cuimo4183** [1Panel-dev/1Panel#6827] [Q&A]
### 1Panel 版本

社区版 v1.10.19-lts

### 问题描述

1. 已在证书页面申请成功证书
2. 在静态网站应用启用HTTPS时保存出错（选择已有证书、手动导入、手动复制粘贴都出现一样错误）
<img width="761" alt="Snipaste_2024-10-22_21-50-22" src="https://github.com/user-attachments/assets/0b60fb5b-eb37-4f65-bbc3-5970b2b12138">

### 重现步骤

<img width="748" alt="Snipaste_2024-10-22_21-54-18" src="https://github.com/user-attachments/assets/53cff2cc-569c-4b33-94d4-a2be8b8a62c5">

**Comment by cuimo4183** [comment]
删除这个应用后，目前连新建静态页面都报错了，完全没法用了
<img width="884" alt="Snipaste_2024-10-22_22-05-30" src="https://github.com/user-attachments/assets/bd4b3d9c-739c-4164-966c-abd90f8c5bf2">

**Comment by zhengkunwang223** [comment]
是不是误删除了网站的 log 目录？

**Comment by cuimo4183** [comment]
没有，我看文件在的，我还给了777权限。最后重新安装openresty才解决
